### ISSUE TITLE
--no-dns ignored, /etc/resolv.conf modified

### ISSUE THREAD

**Issue by Aestu** [adrienverge/openfortivpn#542]
Hi,

Thanks you for this great program.

The problem is _--no-dns_ and _--set-dns=0_ are ignored and my /etc/resolv.conf file is modified. I can reproduce this error in two different machines (my laptop and my raspberry pi):

```
$ more /etc/debian_version 
10.2

$ uname -a
Linux host 4.19.0-6-amd64 #1 SMP Debian 4.19.67-2+deb10u2 (2019-11-11) x86_64 GNU/Linux

$ more /etc/resolv.conf
nameserver 192.168.122.1

$ openfortivpn XXX.XXX.XXX.XXX:443 --no-dns --no-routes -u username -p password --trusted-cert=ddddddddddddddddddddddddddddddddddddddddddddddddddddddddd &
WARN:   You should not pass the password on the command line. Type it interactively or use a config file instead.
WARN:   Bad port in config file: "0".
INFO:   Connected to gateway.
INFO:   Authenticated.
INFO:   Remote gateway has allocated a VPN.
INFO:   Got addresses: [192.168.252.2], ns [10.90.56.68, 10.90.56.69]
INFO:   negotiation complete
INFO:   negotiation complete
INFO:   Interface ppp0 is UP.
INFO:   Tunnel is up and running.

$ more /etc/resolv.conf
nameserver 10.90.56.68
nameserver 10.90.56.69
```

**Comment by DimitriPapadopoulos** [comment]
Which version of _openfortivpn_ is this?

There are multiple options related to DNS:
* `--set-dns` / `--no-dns`: whether _openfortivpn_ rewrites `/etc/resolv.conf`
* `--pppd-use-peerdns` / `--pppd-no-peerdns`: whether _pppd_ rewrites `/etc/resolv.conf`

Have you tried `--no-dns` *and* `--pppd-no-peerdns`?

**Comment by Aestu** [comment]
The version is the one included in Debian Buster repositories: 1.8.1

I didn't see ---pppd-no-peerdns option. This flag is enough to keep my /etc/resolv.conf intact. I don't need to use both flags.

Thank you very much Dimitri
